src/main/kotlin/no/nav/familie/ks/sak/api/BehandlingStegController.kt

```diff
@@ -62,4 +64,46 @@ class BehandlingStegController(
         stegService.utførSteg(behandlingId, BehandlingSteg.BEHANDLINGSRESULTAT)
         return ResponseEntity.ok(Ressurs.success(behandlingService.lagBehandlingRespons(behandlingId = behandlingId)))
     }
+
+    @PutMapping("sett-på-vent")
```

Comment:
Spring prefixer / for oss i disse mapping annotasjonene hvis vi ikke har det med selv.
Men bare for å holde det konsistent så hadde vært fint å legge det inn selv:

```suggestion
    @PutMapping("/sett-på-vent")
```

Reply:
Det gjelder de andre mapping annotasjonene her også.